.eslintrc.js

```diff
@@ -21,23 +21,12 @@ module.exports = {
     // our plugin :D
     //
 
-    '@typescript-eslint/ban-ts-ignore': 'error',
-    '@typescript-eslint/consistent-type-definitions': 'error',
-    '@typescript-eslint/explicit-function-return-type': 'off',
-    '@typescript-eslint/explicit-member-accessibility': 'off',
-    '@typescript-eslint/indent': 'off',
-    '@typescript-eslint/no-explicit-any': 'warn',
-    '@typescript-eslint/no-inferrable-types': 'error',
-    '@typescript-eslint/no-misused-promises': 'error',
+    '@typescript-eslint/consistent-type-definitions': ['error', 'interface'],
+    '@typescript-eslint/no-explicit-any': 'error',
     '@typescript-eslint/no-non-null-assertion': 'off',
```

Comment:
I would love to turn this off, but there is some pain points with nullable properties that mean we have to use them right now. Can consider removing this when the optional chain operator is merged with the next TS version.